src/vm/i386/cgenx86.cpp

```diff
@@ -299,7 +299,12 @@ void TransitionFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 
     MethodDesc * pFunc = GetFunction();
     _ASSERTE(pFunc != NULL);
+
+#ifdef WIN64EXCEPTIONS
+    UpdateRegDisplayHelper(pRD, 0);
```

Comment:
@parjong please ignore my last comment. Reading through the unwinder code more, I've found that it would be difficult to make such a change in a reasonable way.